kubel.el

```diff
@@ -480,18 +480,23 @@ NAME is the buffer name."
   "Return the error buffer name for the PROCESS-NAME."
   (format "*%s:err*" process-name))
 
-(defun kubel--sentinel (process _)
-  "Sentinel function for PROCESS."
-  (let ((process-name (process-name process))
-        (exit-status (process-exit-status process)))
-    (kubel--append-to-process-buffer (format "[%s]\nexit-code: %s" process-name exit-status))
-    (unless (eq 0 exit-status)
-      (let ((err (with-current-buffer (kubel--process-error-buffer process-name)
-                   (buffer-string))))
-        (kubel--append-to-process-buffer (format "error: %s" err))
-        (error (format "Kubel process %s error: %s" process-name err))))))
-
-(defun kubel--exec (process-name args &optional readonly)
+(defun kubel--sentinel (callback)
+  "Sentinel function used by KUBEL--EXEC.
+
+CALLBACK is called when process completes successfully.
+"
+  (lambda (process event)
+    (let ((process-name (process-name process))
+          (exit-status (process-exit-status process)))
+      (kubel--append-to-process-buffer (format "[%s]\nexit-code: %s" process-name exit-status))
+      (if (eq 0 exit-status)
+          (when callback (funcall callback))
+        (let ((err (with-current-buffer (kubel--process-error-buffer process-name)
+                     (buffer-string))))
+          (kubel--append-to-process-buffer (format "error: %s" err))
+          (error (format "Kubel process %s error: %s" process-name err)))))))
+
+(defun kubel--exec (process-name args &optional callback readonly)
```

Comment:
please add a docstring line for `callback`

Reply:
also other nitpick but why did you put `callback` before `readonly`? Does that potentially break backward compatibility in case anyone was using `kubell--exec` directly?

Reply:
> also other nitpick but why did you put `callback` before `readonly`? Does that potentially break backward compatibility in case anyone was using `kubell--exec` directly?

Ah that was an oversight, I think for the backwards compatibility it should be after